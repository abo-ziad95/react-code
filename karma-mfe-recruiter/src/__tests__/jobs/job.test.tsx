import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import Job from "../../views/jobs/job";

it("Job Info renders correctly", () => {
    const job = renderer
        .create(<Karma><Job/></Karma>)
        .toJSON();
    expect(job).toMatchSnapshot();
});