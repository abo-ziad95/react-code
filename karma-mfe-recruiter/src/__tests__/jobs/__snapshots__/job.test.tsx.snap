// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Job Info renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <nav
        className="MuiTypography-root MuiBreadcrumbs-root MuiTypography-body1 MuiTypography-colorTextSecondary"
      >
        <ol
          className="MuiBreadcrumbs-ol"
        >
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Organizations
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Jobs
            </a>
          </li>
          <li
            aria-hidden={true}
            className="PrivateBreadcrumbSeparator-root-144 MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            className="MuiBreadcrumbs-li"
          >
            <p
              className="MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary"
            >
              
            </p>
          </li>
        </ol>
      </nav>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12 MuiGrid-grid-xl-12"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
            <span
              className="MuiTypography-root MuiCardHeader-subheader MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
            >
              <div
                className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
                style={
                  Object {
                    "height": undefined,
                    "width": undefined,
                  }
                }
              />
            </span>
          </div>
          <div
            className="MuiCardHeader-action"
          >
            <div
              className="MuiSkeleton-root MuiSkeleton-circle MuiSkeleton-animate"
              style={
                Object {
                  "height": 48,
                  "marginLeft": 12,
                  "width": 48,
                }
              }
            />
          </div>
        </div>
        <hr
          className="MuiDivider-root"
        />
        <div
          className="MuiCardContent-root"
        >
          <div
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
            style={
              Object {
                "height": undefined,
                "width": undefined,
              }
            }
          />
          <div
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
            style={
              Object {
                "height": undefined,
                "width": undefined,
              }
            }
          />
          <div
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-animate"
            style={
              Object {
                "height": undefined,
                "width": "80%",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
